test/server/utils/addEntries.test.js

```diff
@@ -151,18 +138,14 @@ describe('addEntries util', () => {
     expect(hotClientScript).toEqual(require.resolve(hotClientScript));
   });
 
-  it("should prepends webpack's hot-only client script", () => {
+  it("should prepends webpack's hot only client script", () => {
```

Comment:
```suggestion
  it("should prepend webpack's hot only client script", () => {
```

Reply:
I found a lot of mistakes in test names, let's open issue about it and fix all of them in a separate PR

Reply:
Shall I take it up?

Reply:
Yep